### PR TITLE
CreateCollisionObject

### PR DESCRIPTION
**Pull request by berg0401**
Create a planning scene with BTs @captain-yoshi

### CODE REVIEW

File: src/geometry_msgs.cpp [hunk 1]
```diff
@@ -22,6 +23,7 @@ constexpr auto kPortVector3Stamped = "vector3_stamped";
 constexpr auto kPortTwistStamped = "twist_stamped";
 constexpr auto kPortLinearVelocity = "linear_velocity";
 constexpr auto kPortAngularVelocity = "angular_velocity";
+constexpr auto kPortPose = "pose";
```
Comment (captain-yoshi) [inline src/geometry_msgs.cpp:26]: Put this line before the kPortPoseStamped one.

File: src/set_up_scene.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  std::shared_ptr<moveit_msgs::CollisionObject> object;
+
+  //INPUTS
+  if(!getInput(kPortPlanningSceneInterface, psi) ||
+     !getInput(kPortCollisionObject, object))
+    return NodeStatus::FAILURE;
+  //APPLY OBJECT
+  if(!psi->applyCollisionObject(*object))
+    return NodeStatus::FAILURE;
+  return NodeStatus::SUCCESS;
```
Comment (captain-yoshi) [inline src/set_up_scene.cpp]:
```suggestion
    return NodeStatus::FAILURE;
    
  return NodeStatus::SUCCESS;
```

**Comment by berg0401** [comment]
@captain-yoshi , i changed the box and cylinder dimensions template

**Review by captain-yoshi — APPROVED** [review]
Ok! Next time break down into smaller PR's.

**Pull request merged**